package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dash-core-components",
-  "version": "0.35.2",
+  "version": "0.35.3",
```

Comment:
I like to do minor version updates (e.g. to `0.36.0`) since from the users perspective, this version update adds many new features to the `dcc.Graph` component.